vhd/lib/libvhd.c

```diff
@@ -4381,10 +4387,12 @@ __vhd_io_dynamic_read_bytes(vhd_context_t *ctx,
 			vhd_close(vhd);
 		vhd = &parent;
```

Comment:
Do we still need this line now?

Reply:
hmmm, that's gone weird, I deleted that line and moved it after the open. That should have caused another unused value fail.